ArdusatSDK.cpp

```diff
@@ -10,13 +10,11 @@
 
 #include "ArdusatSDK.h"
 
-RTC_DS1307 RTC;
-
-SdVolume vol;
-#define _output_buffer vol.cacheAddress()->output_buf
-int _output_buf_len = 0;
-
+int OUTPUT_BUF_SIZE = 256;
+int OUTPUT_BUFFER_MAXSIZE = 250;
```

Comment:
I don't like relying on two constants for this sort of thing. If there's some logic that requires a "Max Length" variable that is 6 bytes less than the size of the buffer, that should be calculated on the fly (e.g. `OUTPUT_BUF_SIZE - 6`) instead of having 2 variables to keep track of.